addons/im_livechat/static/src/core/public_web/discuss_channel_model_patch.js

```diff
@@ -34,15 +35,17 @@ const discussChannelPatch = {
         this.shadowedBySelf = 0;
         this.unpinOnThreadSwitch = false;
     },
-    _computeIsDisplayInSidebar() {
-        return this.livechat_status === "need_help" || super._computeIsDisplayInSidebar();
+    /** @override */
+    _computeCanHide() {
+        if (this.livechat_status === "need_help") {
+            return Boolean(this.self_member_id);
+        }
+        return super._computeCanHide(...arguments);
     },
-    get canLeave() {
+    _computeIsDisplayedInSidebar() {
         return (
-            super.canLeave &&
-            (!this.store.discuss.livechatLookingForHelpCategory ||
-                this.store.discuss.livechatLookingForHelpCategory.notEq(this.discussAppCategory) ||
-                this.self_member_id)
+            (!this.self_member_id && this.livechat_status === "need_help") ||
```

Comment:
Note: `need_help` no longer constraining when there is a current member (follow member settings instead)